examples/federated-learning/data_lib.py

```diff
@@ -0,0 +1,214 @@
+"""for simulation only"""
+from collections import OrderedDict
+from functools import wraps
+import logging
+import os
+from pathlib import Path
+
+import tensorflow as tf
+import tensorflow_datasets as tfds
+import tf_encrypted as tfe
+
+logger = logging.getLogger('tf_encrypted')
```

Comment:
maybe we should add this to core TFE, say `tfe.debugging.getLogger`?

Reply:
there is some precedent for doing so from TF, as they have their own debugger module. thinking we should still keep ours separate, but we should also try to standardize how we log things against theirs (currently there are some discrepancies). either way, think that calls for a separate PR :)

Reply:
there is some precedent for doing so from TF, as they have their own debugger module. thinking we should still keep ours separate, but we should also try to standardize how we log things against theirs (currently there are some discrepancies). either way, think that calls for a separate PR :)

Reply:
See https://github.com/tf-encrypted/tf-encrypted/issues/722